cli/args/mod.rs

```diff
@@ -597,6 +595,28 @@ impl CliOptions {
     self.maybe_config_file.as_ref().map(|f| f.specifier.clone())
   }
 
+  // TODO(Cre3per): In deno_ast
+  // - Change MediaType::map_js_like_extension() to operate on extension, not path
+  // - Add MediaType::to_content_type() (This function, but complete)
+  fn media_type_from_file_extension(extension: &str) -> Option<String> {
+    match extension {
+      "ts" => Some("application/typescript"),
+      "tsx" => Some("text/tsx"),
+      "js" => Some("application/javascript"),
```

Comment:
We should use `text/javascript`

https://developer.mozilla.org/ja/docs/Web/HTTP/Basics_of_HTTP/MIME_types

Reply:
Thanks, changed to `text/javascript` and `text/typescript`. I've also linked the relevant deno_ast issue in the TODO comment.

If you're knowledgeable about mime types, you might also have valuable feedback for the deno_ast PR at https://github.com/denoland/deno_ast/pull/134/files